andela/anbu-rc
# #137005177 Digital Products

#### What does this PR do?
This PR adds the ability to sell Digital Products on the application.
#### Description of Task to be completed?
Sell digital goods independently, or mixed with physical goods.
#### How should this be manually tested?
This can be tested by initiating a purchase of a digital product alone or mixed with physical goods and proceed to checkout and payment. On successful order processing, the buyer can see a button to download the product 
#### Any background context you want to provide?
previously we only provide sales of physical goods but with this feature we can sell both physical and digital goods
#### What are the relevant pivotal tracker stories?
#137005177
#### Screenshots (if appropriate):
![digital-product](https://cloud.githubusercontent.com/assets/21035558/23281391/6e3e8956-fa1d-11e6-8d97-fb4757e2aaef.png)
#### Questions:
No questions
